src/screens/ForgotPassword/forgotPasswordHandler.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import AuthAdapter from 'adapters/authAdapter';
+
+const ForgotPasswordHandler = () => {
+  const [requestSuccess, setRequestSuccess] = useState(false);
+  const [error, setError] = useState(null);
+
+  const handleSubmit = async (values) => {
+    try {
+      await AuthAdapter.forgotPassword(values.email).then(function (response) {
```

Comment:
@rafayet-monon How about using the arrow function here too?
```suggestion
      await AuthAdapter.forgotPassword(values.email).then(response => {
```